src/plugins/dam/components/image-picker-field/LinkedImages.jsx

```diff
@@ -19,42 +19,63 @@ function LinkedImages(props) {
     request.set('linked_ref', NodeRef.fromString(nodeRef));
   }, [nodeRef, request]);
 
-  return(
+  return (
     <>
       {(!response || pbjxError) && <Loading error={pbjxError} />}
       {response && response.has('nodes') && (
         <div className="image-grid-container">
           <Row className="gx-3">
             {response.get('nodes').map(node => {
-                return (
-                  <Col key={node.get('_id')} xs={12} sm={6} md={4} lg={3} className="col-xl-2p">
-                    <Card
-                      className="p-2 card-hover-border card-shadow text-white"
-                      onClick={() => {
-                        selectImage(node.generateNodeRef());
-                        toggle();
-                      }}
-                    >
-                      <Media className="aspect-ratio aspect-ratio-1by1">
-                        <Media
-                          object
-                          src={damUrl(node, '1by1', 'sm')}
-                          alt="thumbnail"
-                          width="200"
-                          height="200"
-                        />
-                        <CardImgOverlay>
-                          <CardTitle className="h5 mb-0">{node.get('title')}</CardTitle>
-                        </CardImgOverlay>
-                      </Media>
-                    </Card>
-                  </Col>
-                );
-              }
+              return (
```

Comment:
@meghedi nit: can just use the implicit return

Reply:
Done.